app/editor/src/components/form/text/FormSizing.ts

```diff
@@ -0,0 +1,7 @@
+export enum FormInputSizes {
```

Comment:
I thought we were using the `FieldSize`?

Reply:
Right we are! Need to remove this. Snuck in on my other branch